### ISSUE TITLE
android fullscreen and autoplay

### ISSUE THREAD

**Issue by foliovision** [foliovision/freedom-video-player#447]
Original issue link: [#614 android fullscreen and autoplay](https://github.com/flowplayer/flowplayer/issues/614)

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/90533?v=4' /> [sleith](https://github.com/sleith) commented on Dec 28, 2013

Hi, i've been trying to make the video autoplay and full screen. But i don't have luck yet. Is it possible for flowplayer to do so on android?
Here's what i load with webview:

<html>

```
<head>

    <link
        href="file:///android_asset/skin/minimalist.css"
        rel="stylesheet"
        type="text/css" >

        <style type="text/css" >
```

body { font: 12px "Myriad Pro", "Lucida Grande", sans-serif; text-align: center; padding-top: 5%; }.flowplayer { width: 100%; height: 100%; }
            </style>

```
        <script
            src="http://ajax.googleapis.com/ajax/libs/jquery/1/jquery.min.js"
            type="text/javascript" >
        </script>

        <script
            src="file:///android_asset/flowplayer.min.js"
            type="text/javascript" >
        </script>
    </link>
</head>

<body>

    <div
        class="flowplayer"
        data-ratio="0.4167" >

        <video autoplay="autoplay" >

            <source
                height="100%"
                src="http://stream.flowplayer.org/bauhaus/624x260.mp4"
                type="video/mp4"
                width="100%" >
            </source>
        </video>
    </div>
</body>
```

</html>

Thanks

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/168338?v=4' /> [phloxic](https://github.com/phloxic) commented on Dec 28, 2013

Please ask usage questions in the forum: http://flowplayer.org/forum/ not in the bug tracker. Thank you.
Remember:
Not all devices allow autoplay.
Do not specify dimensions for the source tag, and, with Flowplayer, not even for the video tag. Instead have a look at this demo: http://flowplayer.org/demos/huge/ - includes link to standalone demo for copying and tweaking.
